src/app/bot.py

```diff
@@ -106,6 +106,35 @@ def menu_actions(update: Update, context: CallbackContext) -> None:
     ))
 
 
+def all_callback(update: Update, message_text: str, reply_markup: InlineKeyboardMarkup) -> None:
```

Comment:
I am not sure why this function is called `all_callback` when it is not used on all actual callbacks. 
Maybe we should rename it with something else ?
```suggestion
def sendMarkupAnswer(update: Update, message_text: str, reply_markup: InlineKeyboardMarkup) -> None:
```

Reply:
yea thanks, i just did that